proyecto/src/main/java/com/proyecto/proyecto/ControladorUsuarios.java

```diff
@@ -33,17 +33,17 @@ public String eliminar(@PathVariable("id") int idUsuario, RedirectAttributes att
     }
 
     @GetMapping("/unlock/{id}")
-	public String activar(@PathVariable("id") int idUsuario, RedirectAttributes attributes) {		
-    	servicioUsuarios.activar(idUsuario);
-		attributes.addFlashAttribute("msg", "El usuario fue activado y ahora tiene acceso al sistema.");		
-		return "redirect:/usuarios/index";
+	public String activar(@PathVariable("id") int idUsuario, RedirectAttributes attributes) {
```

Comment:
#should Líneas 36, 40, 43 y 47 tienen que ir también un pasito para atrás en la indentación